orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -571,6 +572,11 @@ class OWPreprocess(OWWidget):
     persistent_data_filter = settings.Setting({})
     preprocessors = []  # Pre-processing modules for the current run.
 
+    UserAdviceMessages = [
+        widget.Message(
+            "NLTK data was downloaded to: {}".format(Downloader().default_download_dir()),
```

Comment:
Could you make it just a bit more friendly? Like: "Some preprocessing methods require data (like word relationships, stop words, punctuation rules etc.) from the NLTK package. This data, if you didn't have it already, was downloaded to: {}"